config/smarty.config.inc.php

```diff
@@ -88,6 +88,7 @@
 smartyRegisterFunction($smarty, 'function', 'widget', 'smartyWidget');
 smartyRegisterFunction($smarty, 'block', 'widget_block', 'smartyWidgetBlock');
 smartyRegisterFunction($smarty, 'modifier', 'classnames', 'smartyClassnames');
+smartyRegisterFunction($smarty, 'function', 'getUrl', array('Link', 'getUrlSmarty'));
```

Comment:
ps_url, we talking about it on gitter and i remember that we condult that it will be good to have global prefix like in ps_widget :)